docs/api_specification.md

```diff
@@ -339,3 +339,39 @@ Content-Type: application/json
 | `amount`                 | X              | amount of the payment in pence. |
 | `_links.self`            | X              | self GET link to the payment |
 | `_links.next`            | X              | next GET link |
+
+## PATCH /v1/api/gateway-account/{gatewayAccountId}
+
+This endpoint batch updates Service Names of Products with a given gatewayAccountId
+
+### Request example
+
+```
+PATCH /v1/api/gateway-account/56
+Authorization: Bearer API_TOKEN
+Content-Type: application/json
+
+{
+    "op"    :     "replace",
```

Comment:
Why we need to specify and operation? Isn't PATCH always about updating? For deleting we have the DELETE method.

Reply:
it could be `disable`, `enable`, etc